src/integer_mod_q/poly_over_zq/from.rs

```diff
@@ -17,6 +17,44 @@ use crate::{error::MathError, integer::PolyOverZ, integer_mod_q::modulus::Modulu
 use flint_sys::fmpz_mod_poly::{fmpz_mod_poly_init, fmpz_mod_poly_set_fmpz_poly};
 use std::{mem::MaybeUninit, str::FromStr};
 
+impl From<(&PolyOverZ, &Modulus)> for PolyOverZq {
+    /// Create a [`PolyOverZq`] from a [`PolyOverZ`] and [`Modulus`].
+    ///
+    /// Parameters:
+    /// - `poly_modulus_tuple`: A tuple of the polynomial and the modulus.
```

Comment:
previously you used a different notation for tuples. Please make this consistent (and add it to the next refactor, for other places it has tobe fixed in the docs)